src/Tuura/Plato/Translation.hs

```diff
@@ -86,15 +86,12 @@ cartesianProduct l = removeSupersets sortAllLists
   where
     sequenced    = sequence (NonEmpty.toList l)
     removeDupes  = map nub sequenced
```

Comment:
If I remove this line, it goes back to having many redundancies in a translated STG.
It is there because, if you combine `[a-, c-], [a-,d-], [b-,c-], [b-,d-]`
Then you get results such as:
`[a-, a-, b-, b-], [a-, a-, b-, d-], [a-, a-, c-, b-]...`
This serves to remove the duplicates within each list